### ISSUE TITLE
Option to tell robot not to create the html & xml logs 

### ISSUE THREAD

**Issue by damies13** [damies13/rfswarm#151]
**Is your feature request related to a problem? Please describe.**
having more options to manage disk usage and time to download logs at end of test

**Describe the solution you'd like**
Add and option to tell robot not to create the html & xml logs to the scenario

**Comment by damies13** [comment]
While not obvious this is mostly possible to do already as mentioned in this post --> https://github.com/damies13/rfswarm/issues/147#issuecomment-1562656893

This feature request is to have configuration options shown in the settings page to disable/enable the logs.

I'm not sure if this should be a global option for all script or a per script option or both. anyone want to add an opinion.

**Comment by damies13** [comment]
Relevant options:
```
-o NONE -l NONE -r NONE
```

```
 -o --output file         XML output file. Given path, similarly as paths given
                          to --log, --report, --xunit, and --debugfile, is
                          relative to --outputdir unless given as an absolute
                          path. Other output files are created based on XML
                          output files after the test execution and XML outputs
                          can also be further processed with Rebot tool. Can be
                          disabled by giving a special value `NONE`.
                          Default: output.xml

 -l --log file            HTML log file. Can be disabled by giving a special
                          value `NONE`. Default: log.html
                          Examples: `--log mylog.html`, `-l NONE`
 -r --report file         HTML report file. Can be disabled with `NONE`
                          similarly as --log. Default: report.html

```

**Comment by damies13** [comment]
Agent and manager parts done, now need to update documentation and add test cases

**Comment by damies13** [comment]
Issue #151 merged into release branch v1.3.0
